scripts/install.sh

```diff
@@ -22,21 +22,7 @@ fi
 
 if command_exists socketplane; then
     echo >&2 'Warning: "socketplane" command appears to already exist.'
-    while true; do
-        read -p "Would you like to re-install socketplane (y/n) " yn
-        case $yn in
-            [Yy]* )
-                cleanup
-                break
-                ;;
-            [Nn]* )
-                exit 1
-                ;;
-            * )
-                echo "Please answer yes or no."
-                ;;
-        esac
-    done
+    echo >&2 'Warning: "Please run "socketplane uninstall first'
```

Comment:
It was https://github.com/socketplane/socketplane/issues/35
My suggestion is this...
If socketplane is already installed, prompt the user that reinstall will proceed in 20 seconds unless they issue a CTRL+C, otherwise, run the "cleanup" function and continue execution